### ISSUE TITLE
Reactotron UI requires jumping between tabs often, and can be improved

### ISSUE THREAD

**Issue by sballew** [infinitered/reactotron#1081]
Reactotron makes it difficult to see all needed information at the same time. I frequently jump from the Timeline tab, looking at log messages and network requests, to viewing the State tab to view MST data. Similarly, the Timeline tab combines log and network data into a single timeline, which can make it difficult to process that data at a glance.

When comparing this to react-native-debugger (RND), RND shows these different data sections as separate windows within the same view. The benefit of this is you can view your store data, your log messages, and your network requests all in an organized and side-by-side method.

I believe Reactotron could benefit from implementing a similar UI, where a developer can see all of the usual information at a glance from a single view with less context switching.

**Comment by rmevans9** [comment]
This is a great idea I think actually. Thinking back to the roots of reactotron back before it was a GUI based application it was a terminal application that had little sections for each "tab" as it is now. I have been thinking a lot about the UI structure of the reactotron app and have been starting to feel the same way with having to jump between tabs. I like this idea a lot. I am just wondering if there should be a static configuration of the data or if allowing some level of customization would make more sense.

**Comment by GrinZero** [comment]
I completely agree that the design of Chrome Devtool has become a user habit for web developers and has also passed community validation. I think we can refer to its design to build a GUI. For example, turning some filters into tabs:
<img width="1019" alt="image" src="https://github.com/user-attachments/assets/16165878-304e-4b42-94c1-df96988fc697">

**Comment by GrinZero** [comment]
This is my first day using `Reactotron`, and I think it's a really great project, but its UI is a bit messy. For example, when I tried to add the `MMKV` plugin, I could only find it in the timeline.
And my expectation is to have an independent tab.

**Comment by GrinZero** [comment]
like this:
<img width="952" alt="image" src="https://github.com/user-attachments/assets/589f467f-3a62-4602-983e-f14371590464">
